internal/pkg/model/file.go

```diff
@@ -0,0 +1,86 @@
+package model
+
+import (
+	"fmt"
+
+	"github.com/iancoleman/orderedmap"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/json"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils"
+)
+
+type FileLine struct {
+	Line   string
+	Regexp string
+}
+
+// FilePath relative to the project dir.
+type FilePath string
+
+type File struct {
+	Content string
+	Desc    string
+	Path    FilePath
+}
+
+type JsonFile struct {
+	Content *orderedmap.OrderedMap
+	Desc    string
+	Path    FilePath
+}
+
+// ObjectFiles - all files related to the object, when saving.
+type ObjectFiles struct {
```

Comment:
Bude pouzite v dalsich PR, .... stuktura, ktora obsahuje vsetky subory naviazane na objekt.
Bude sa to pouzivat pri load/save transfomacii a shared codes.